README.md

```diff
@@ -0,0 +1,60 @@
+# Beginning Web Development 
+(TODO: Add brief description of this repo here)
+
+**NOTE TO DEVS**: if you're working in VSCode, reading the following might be hard and you wouldn't see the embedded images. Click the indicated button to open preview for a cleaner format (usually at the upper right corner of the screen, it's a square icon with a magnifying glass at its bottom right):
+![square icon with a magnifying glass at its bottom right](images/readmeButton.png)
+
+## Important Links
+* [Figma Mockups](https://www.figma.com/design/8GS4B4rOXS7OKaY5H4g2br/BWD-Design-Project?node-id=0-1&t=uDAFUoXuth5RSgD7-0)
+
+
+## Working with Git and GitHub:
+
+1. Clone the repo via HTTPS, and switch to "develop":
+```sh
+git clone https://github.com/beginning-web-dev/bwd-frontend  # pull remote repo to local
+cd bwd-frontend # navigate to work directory
+git switch develop # switch from main (default) to develop branch
+```
+2. Making changes and pushing branch:
+```sh
+git branch bwd-5 # create a new branch and name it your ticket number
+
+# work on your task then stage, commit, and push your work...
+
+git add <file1> <file2> # add the files you want to stage, alternatively you could do 'git add .' to add all modified files
```

Comment:
You could word wrap the code block comments at 95 characters to make it more readable. You have to scroll to the right to see the rest of the comment, initially leading me to believe this line ended at "alternatively you could do 'git add'".
![image](https://github.com/user-attachments/assets/972bd050-3bc7-4cbf-8fdd-1ff2831cb429)